101-monitoring-dynamic-metric-alert/azuredeploy.json

```diff
@@ -46,6 +46,7 @@
     },
     "metricName": {
       "type": "string",
+      "defaultValue": "Percentage CPU",
```

Comment:
remove the parameter value from the parameter file so this defaultValue is tested